client/my-sites/site-settings/general/main.jsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import DocumentHead from 'components/data/document-head';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+import SiteSettingsNavigation from 'my-sites/site-settings/navigation';
+import GeneralForm from 'my-sites/site-settings/form-general';
+import DeleteSiteOptions from 'my-sites/site-settings/delete-site-options';
+import QuerySitePurchases from 'components/data/query-site-purchases';
+import config from 'config';
+import notices from 'notices';
+import { getSitePurchases, hasLoadedSitePurchasesFromServer, getPurchasesError } from 'state/purchases/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+class SiteSettingsGeneral extends Component {
+	static propTypes = {
+		site: PropTypes.object.isRequired,
+		hasLoadedSitePurchasesFromServer: PropTypes.bool.isRequired,
+		sitePurchases: PropTypes.array.isRequired,
+		purchasesError: PropTypes.object,
+	};
+
+	componentWillReceiveProps( nextProps ) {
+		if ( nextProps.purchasesError ) {
+			notices.error( nextProps.purchasesError );
+		}
+	}
+
+	render() {
+		const {
+			site,
+			sitePurchases,
+			translate,
+		} = this.props;
+
+		return (
+			<Main className="general__main site-settings">
```

Comment:
I'd certainly love to, but not right now. It currently contains a lot of the styles within site settings, so I suggest that we leave that for another PR.